run.py

```diff
@@ -32,24 +32,38 @@
 ALL_SCRIPTS = SENSOR_SCRIPTS + SOURCE_SCRIPTS + REPORT_SCRIPTS
 
 
-def _run_script(script, config, root_dir):
+def _run_script(script, config, root_dir, task, session, run, subject):
     logger.info(f'Now running: {script}')
 
     # It's okay to fiddle with the environment variables here as process()
     # has set up some handlers to reset the environment to its previous state
     # upon exit.
     env = os.environ
     env['MNE_BIDS_STUDY_CONFIG'] = str(pathlib.Path(config).expanduser())
+
     if root_dir:
         env['BIDS_ROOT'] = str(pathlib.Path(root_dir).expanduser())
 
+    if task:
+        env['MNE_BIDS_STUDY_TASK'] = task
+
+    if session:
+        env['MNE_BIDS_STUDY_SESSION'] = session
+
+    if run:
+        env['MNE_BIDS_STUDY_RUN'] = run
+
+    if subject:
+        env['MNE_BIDS_STUDY_SUBJECT'] = subject
+
     module_name = script.replace('.py', '')
     runpy.run_module(mod_name=module_name, run_name='__main__')
 
     logger.info(f'Successfully finished running: {script}')
 
 
-def process(steps, config, root_dir=None):
+def process(steps, config, root_dir=None, task=None, session=None,
+            run=None, subject=None):
```

Comment:
I'm not sure about the order of params here. Wonder if it would make more sense to kind of follow the "BIDS order"? i.e. subject, session, task, run? WDYT?